generated/lspconfig-servers.json

```diff
@@ -778,6 +783,13 @@
     "desc": "    https://github.com/lexical-lsp/lexical\n\n    Lexical is a next-generation language server for the Elixir programming language.\n\n    Follow the [Detailed Installation Instructions](https://github.com/lexical-lsp/lexical/blob/main/pages/installation.md)\n\n    **By default, `lexical` doesn't have a `cmd` set.**\n    This is because nvim-lspconfig does not make assumptions about your path.\n    ",
     "name": "lexical"
   },
+  {
+    "cmd": ["lsp-ai"],
+    "desc": [
```

Comment:
Got created as a list of strings for some reason

Reply:
They defined it as a list upstream, idk why though (https://github.com/neovim/nvim-lspconfig/blob/9b2509f17c284486497358ccea1019cc17c28af6/lua/lspconfig/configs/lsp_ai.lua#L15-L28)

Reply:
Thanks for checking, do we want to create a workaround to flatten lists for that or try and upstream a change?

Reply:
Added a comment https://github.com/neovim/nvim-lspconfig/pull/3206/files#r1801604783 to double check

Reply:
> Thanks for checking, do we want to create a workaround to flatten lists for that or try and upstream a change?

It would be best to fix the issue upstream, but this could take a while to trickle back in nixvim, so we may still need a work-around (we could manually edit the generated file if we upstream sufficiently quickly a "fix")

Reply:
We can also forego running the generate task when doing update PRs, assuming there's no major changes that it would otherwise produce